### PR TITLE
Add What is the Internet video for CSP

### PR DESCRIPTION
**Pull request by sfilman**

### CODE REVIEW

**Comment by sfilman** [comment]
@Bjvanminnen sanity check plz :)

File: dashboard/config/videos.csv [hunk 1]
```diff
@@ -126,6 +126,7 @@ csp_concept_bigdata_medicine			bMrDHtGHFR4	https://videos.code.org/2015/csp/conc
 csp_concept_cybercrime			AuYNXgO_f3Y	http://videos.code.org/2015/csp/concept_cybercrime.mp4
 csp_concept_encryption			ZghMPWGXexs	http://videos.code.org/2015/csp/concept_encryption.mp4
 csp_concept_http_html			kBXQZMmiA4s	http://videos.code.org/2015/csp/concept_http_html.mp4
+csp_concept_what_is_internet			Dxcc6ycZ73M	https://videos.code.org/2015/csp/what_is_the_internet.mp4
```
Comment (Bjvanminnen) [inline dashboard/config/videos.csv:129]: all of our other videos use http instead of https here. not sure it matters, but also not sure it doesnt

Comment (Bjvanminnen) [inline dashboard/config/videos.csv:129]: actually, as i expand the diff i see we already have a mix and this is prob fine

**Comment by Bjvanminnen** [comment]
lgtm

**Comment by jeremydstone** [comment]
Reverted because the video addition caused all video thumbnails to shift by one

**Pull request merged**